course/lesson-11-time-delta/lesson.tres

```diff
@@ -96,11 +96,11 @@ There will always be milliseconds variations from frame to frame.
 
 That is why the [code]_process()[/code] function receives a [code]delta[/code] parameter.
 
-The word \"delta\" represents a time difference. It's the time passed since the previous frame, in seconds.
+Delta represents a time difference. It's the time passed since the previous frame, in seconds.
 
 The [code]delta[/code] parameter tells us how long it took for Godot to complete the [b]previous frame[/b].
 
-We can use it to ensure that the variations between frames don't make the game's simulation unreliable."
+We can use it to ensure that the changes between frames don't make the game's behaviour unreliable. This is because different computers run differently, so a fast computer will have more frames per second than a slow computer. If we ignore [code]delta[/code] then the game experience will vary, depending on the computer. Delta helps to make the game experience consistent for everyone."
```

Comment:
Please use US English for consistency (behaviour -> behavior).

Also, avoid making paragraphs on a single line; mostly for consistency in looks with the rest of the app.

```suggestion
We can use it to ensure that the changes between frames don't make the game's behavior unreliable.

This is because different computers run differently, so a fast computer will have more frames per second than a slow computer.

If we ignore [code]delta[/code], the game experience will vary, depending on the computer. Delta helps to make the game experience consistent for everyone."
```